### PR TITLE
Mileston 3: Portfolio Project Mobile version Contact section

### PR DESCRIPTION
**Pull request by Profsain**
- Added contact section with HTML structure
- Added CSS style for contact section by Richmond
- Added Form button and updated Readme file by Vishal

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,173 +83,213 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                placeholder="Enter Text Here"
+              ></textarea>
+              <button class="section-btn" type="submit" id="submit">
+                Get in Touch
+              </button>
+            </form>
```
Comment (m05e5) [inline index.html]: GG with your form.
-  [ ] As part of the requirements and to ensure a random user does not submit an empty form, I recommend adding `required` to each of your input field

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Profsain , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

-  [ ] As part of the requirements and to ensure a random user does not submit an empty form, I recommend adding `required` to each of your input field

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -83,173 +83,216 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+              class="form"
+              method="post"
+            >
+              <input
+                type="text"
+                name="user_name"
+                id="user_name"
+                placeholder="Full name"
+                required
+              />
```
Comment (thecodechaser) [inline index.html]: - [ ] The name field should accept up to 30 chars. Please make sure the name field does not accept more than 30 characters. You can use this attribute `maxlength="30"`.

File: index.html [hunk 3]
```diff
@@ -83,173 +83,216 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+              <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                placeholder="Enter Text Here"
+                required
+              ></textarea>
```
Comment (thecodechaser) [inline index.html]: - [ ] The message field should accept up to 500 chars. Please make sure the message field does not accept more than 500 characters. You can use this attribute `maxlength="500"`.

File: index.html [hunk 4]
```diff
@@ -83,173 +83,216 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
           </li>
         </ul>
       </section>
+      <section class="inner-section" id="contactSection">
+        <div class="form-container">
+          <div class="contact-intro">
+            <p class="section-title">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </p>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please use the correct font size for the form paragraph and make sure it matches with the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design

image

### Figma design

image

File: index.html [hunk 5]
```diff
@@ -83,173 +83,216 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+              action="https://formspree.io/f/xjvlpydr"
+              class="form"
+              method="post"
+            >
+              <input
+                type="text"
+                name="user_name"
+                id="user_name"
+                placeholder="Full name"
+                required
```
Comment (thecodechaser) [inline index.html:274]: - [ ] Please place some padding from the left for all form fields and make sure it matches with the Figma template.

### Your design

image

### Figma design

image

File: index.html [hunk 6]
```diff
@@ -83,173 +83,216 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+              ></textarea>
+              <button class="section-btn" type="submit" id="submit">
+                Get in Touch
+              </button>
+            </form>
+          </div>
+        </div>
+      </section>
     </main>
   </body>
```
Comment (thecodechaser) [inline index.html]: - [ ] Your design is missing social media icons and the email address from the Figma template. Please make sure to add them to your design

### See the image for reference 

image

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Profsain👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Optional suggestions
I am marking these changes as optional because they were missed by the previous reviewer but I strongly suggest considering them.
- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**